### PR TITLE
Graham stealing the initial commit, its mine now

### PR DESCRIPTION
**Pull request by Raareus**
Change to comment about the mouse being on screen for clarity.

### CODE REVIEW

File: Tic Tac Toe/scenes/main.gd [hunk 1]
```diff
@@ -32,7 +32,7 @@ func _process(delta):
 func _input(event):
 	if event is InputEventMouseButton:
 		if event.button_index == MOUSE_BUTTON_LEFT and event.pressed:
-			#check if mouse is on the game board
+			#check if mouse is on the game board, if it is you a bitch
```
Comment (rocchida) [inline Tic Tac Toe/scenes/main.gd:35]: Im sorry this is really great, honest

**Comment by CyborgJiro** [comment]
what a great addition to my code

File: Tic Tac Toe/scenes/main.gd (hunk 1 above)
Comment (CyborgJiro) [inline Tic Tac Toe/scenes/main.gd:35]: damn dude, nice change

**Review by rocchida — APPROVED** [review]

File: Tic Tac Toe/scenes/main.gd (hunk 1 above)
Comment (Raareus) [reply Tic Tac Toe/scenes/main.gd:35]: a q p
    w

**Pull request merged**